test/forge/ERC4626Test.sol

```diff
@@ -104,7 +105,7 @@ contract ERC4626Test is IntegrationTest, IMorphoFlashLoanCallback {
         uint256 redeemed = vault.withdraw(withdrawn, RECEIVER, ONBEHALF);
 
         assertEq(vault.balanceOf(ONBEHALF), shares - redeemed, "balanceOf(ONBEHALF)");
-        assertEq(vault.idle(), deposited - withdrawn, "idle");
+        assertEq(_idle(), deposited - withdrawn, "idle");
```

Comment:
why do we keep this test first?

Reply:
I don't understand the question: it's a useful test isn't it?